feature/home/src/main/java/com/guidal/feature/home/main/MainScreen.kt

```diff
@@ -1,43 +1,108 @@
+        HomeNavigationButtonModel("Shops", "Post", onClick = {}, sectionIcon = UiModelMenuButtonIcon(imageVector = GuidalIcons.Default.Shop)),
+        HomeNavigationButtonModel("Trails", "Post", onClick = {}, sectionIcon = UiModelMenuButtonIcon(imageVector = GuidalIcons.Default.Hiking)),
+        HomeNavigationButtonModel("Must Visit", "Category", onClick = {}, sectionIcon = UiModelMenuButtonIcon(imageVector = GuidalIcons.Default.CircledStar)),
+        HomeNavigationButtonModel("Sightseeing", "Category", onClick = {}, sectionIcon = UiModelMenuButtonIcon(imageVector = GuidalIcons.Default.Museum)),
+        HomeNavigationButtonModel("Restaurants", "Category", onClick = {}, sectionIcon = UiModelMenuButtonIcon(imageVector = GuidalIcons.Default.Restaurant)),
+        HomeNavigationButtonModel("Beaches", "Category", onClick = {}, sectionIcon = UiModelMenuButtonIcon(imageVector = GuidalIcons.Default.Beach)),
+        HomeNavigationButtonModel("Night Life", "Category", onClick = {}, sectionIcon = UiModelMenuButtonIcon(imageVector = GuidalIcons.Default.NightLife)),
+        HomeNavigationButtonModel("Favorites", "Category", onClick = {}, sectionIcon = UiModelMenuButtonIcon(imageVector = GuidalIcons.Outlined.Favorite))
+    )
+
     Scaffold(
+        topBar = {
+            TopAppBar(
+                title = "Patras",
+                navigationIcon = UiModelTopAppBarIcon(
+                    icon = GuidalIcons.Default.Guidal,
+                    onClick = { },
+                    color = MaterialTheme.colorScheme.primary,
+                    isEnabled = false
+                ),
+                modifier = Modifier
+                    .padding(horizontal = 4.dp)
+            )
+        },
         modifier = modifier
     ) { innerPadding ->
         Column(
             modifier = modifier
                 .fillMaxSize()
                 .padding(innerPadding)
-                .padding(horizontal = 20.dp)
+                .padding(horizontal = 10.dp)
         ) {
-            // TODO Remove
-            Row(
+
+            // TODO: REMOVE AFTER WEATHER IMPLEMENTATION
+            // Date section
+            val currentDate = LocalDate.now()
+            val dayOfWeek = currentDate.dayOfWeek.getDisplayName(TextStyle.FULL, Locale.getDefault())
+            val formattedDate = currentDate.format(DateTimeFormatter.ofPattern("yyyy-MM-dd"))
+
+            Column(
                 modifier = Modifier
                     .fillMaxWidth()
-                    .padding(vertical = 20.dp),
-                horizontalArrangement = Arrangement.Center
+                    .padding(top = 20.dp),
+                horizontalAlignment = Alignment.CenterHorizontally
             ) {
                 Text(
-                    text = "Home Screen",
-                    style = MaterialTheme.typography.titleLarge
+                    color = MaterialTheme.colorScheme.onSurfaceVariant,
+                    style = MaterialTheme.typography.titleMedium,
```

Comment:
Use `.labelLarge` or `.labelMedium`. `.titleSize` is for really big texts, `.bodySize` is for texts without bold and `.labelSize` is for texts with bold.